### PR TITLE
Markdown examples update

### PR DESCRIPTION
**Pull request by mikikiv**
Example output of the combining of multiple Markdown components was incorrectly inside a code block when it should be rendered as Markdown

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@mikikiv: Looks good on first review. Handing off for review number two.

File: content/markdown/concepts/emphasis/emphasis.md [hunk 1]
```diff
@@ -41,16 +41,16 @@ To italicize text, add one asterisk `*` or underscore `_` before and after a wor
...
-
 For example:
 
 ```markdown
 This avacado egg sandwich is _very_ good.
 ```
 
+The output would be:
+
+This avacado egg sandwich is _very_ good.
```
Comment (yangc95) [inline content/markdown/concepts/emphasis/emphasis.md:52]:
```suggestion
This avocado egg sandwich is _very_ good.
```

🥑

Comment (yangc95) [reply content/markdown/concepts/emphasis/emphasis.md:52]: @mikikiv This is the only other thing noticed 😆

**Comment by Dusch4593** [comment]
@mikikiv I'm gonna take one last look at this PR before we merge. 😄

File: content/markdown/concepts/emphasis/emphasis.md [hunk 2]
```diff
@@ -23,16 +23,16 @@ To bold text, add two asterisks `**` or underscores `__` before and after a word
 **bold text**
 ```
 
-```markdown
-**bold text**
-```
-
 For example:
 
 ```markdown
```
Comment (Dusch4593) [inline content/markdown/concepts/emphasis/emphasis.md:28]: Could we change all instances of \```markdown to \```md, per the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#code-blocks)?

File: content/markdown/concepts/emphasis/emphasis.md [hunk 3]
```diff
@@ -23,16 +23,16 @@ To bold text, add two asterisks `**` or underscores `__` before and after a word
...
 Please do **NOT** touch your pimples.
 ```
 
+Will render as:
```
Comment (Dusch4593) [inline content/markdown/concepts/emphasis/emphasis.md:32]:
```suggestion
The output would be:
```

File: content/markdown/concepts/emphasis/emphasis.md [hunk 4]
```diff
@@ -41,16 +41,16 @@ To italicize text, add one asterisk `*` or underscore `_` before and after a wor
...
+This avocado egg sandwich is _very_ good.
 ```
 
+The output would be:
+
+This avocado egg sandwich is _very_ good.
+
 ## Strikethrough
 
 To make a strikethrough text, use 2 tildes `~` before and after a piece of text.
```
Comment (Dusch4593) [inline content/markdown/concepts/emphasis/emphasis.md:56]:
```suggestion
A strikethrough can be applied to text by surrounding with two tildes `~~`, like in the example below:
```

And then I would consolidate the following two snippets into one that looks like this:

```md
I did ~~not~~ build a website.
```

File: content/markdown/concepts/blockquotes/blockquotes.md [hunk 5]
```diff
@@ -67,12 +67,10 @@ Blockquotes can contain other Markdown formatted elements. Not all elements can
...
 The output would be:
 
-```markdown
 > ### The quarterly results look great
 >
 > - Dogecoin was off the charts!
 > - Apes together stonk!
 > - $AMC to the moon!
 >
 >   _Everything_ is going according to **the plan**.
```
Comment (Dusch4593) [inline content/markdown/concepts/blockquotes/blockquotes.md:76]:
```suggestion
> _Everything_ is going according to **the plan**.
```

Comment (Dusch4593) [reply content/markdown/concepts/blockquotes/blockquotes.md:76]: Let's also fix that indent above on line # 65.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@mikikiv Finished with my review. Really appreciate these solid edits! 🙏 

In addition to the comments/suggestions I left for you to check out, I'll also post some high-level comments here:

- Let's remove any/all instances of "you" and "we" voices (such as on line # 56 in `blockquotes.md`).
- Below line # 51 in `blockquotes.md`, can we include output?
- Replace all instances of \```markdown with \```md.

**Comment by Dusch4593** [comment]
Resolved some conflicts in 9eef4d63639488d61a44db3d83b2586d6c76aae8 by incorporating changes from #667 (now merged).

**Comment by Dusch4593** [comment]
Hey @mikikiv! 👋🏻 Just wanted to check in and see how everything is going on your side of things? I'm happy to help in any way I can 😄

File: content/markdown/concepts/blockquotes/blockquotes.md (hunk 5 above)
Comment (yangc95) [reply content/markdown/concepts/blockquotes/blockquotes.md:76]: Oh, yeah, we should also remove the italic underscores as per Products Team

Comment (yangc95) [reply content/markdown/concepts/blockquotes/blockquotes.md:76]: whoops please disregard my last comment for this case

File: content/markdown/concepts/emphasis/emphasis.md (hunk 2 above)
Comment (mikikiv) [reply content/markdown/concepts/emphasis/emphasis.md]: Totally! Done in [b03c375](https://github.com/Codecademy/docs/pull/656/commits/b03c375e2e651107a2a9fd390c12e4c6a12f7efb)

File: content/markdown/concepts/blockquotes/blockquotes.md [hunk 6]
```diff
@@ -17,62 +17,60 @@ Blockquotes indicate that the enclosed text is an extended quotation. They can a
...
-Blockquotes can contain other Markdown formatted elements. Not all elements can be used — you’ll need to experiment to see which ones work.
+Blockquotes can contain other Markdown formatted elements, though, not all elements can be used — you’ll need to experiment to see which ones work with your .md processor.
 
-```markdown
+```md
 > ### The quarterly results look great
 >
 > - Dogecoin was off the chart!
 > - Apes together stonk!
 > - $AMC to the moon!
```
Comment (Dusch4593) [inline content/markdown/concepts/blockquotes/blockquotes.md:63]:
```suggestion
> - Sales were off the chart!
> - Morale is strong!
> - Poised to go public next spring!
```

File: content/markdown/concepts/blockquotes/blockquotes.md [hunk 7]
```diff
@@ -17,62 +17,60 @@ Blockquotes indicate that the enclosed text is an extended quotation. They can a
...
 ```
 
 The output would be:
 
-```markdown
 > ### The quarterly results look great
 >
 > - Dogecoin was off the charts!
 > - Apes together stonk!
 > - $AMC to the moon!
```
Comment (Dusch4593) [inline content/markdown/concepts/blockquotes/blockquotes.md:74]:
```suggestion
> - Sales were off the chart!
> - Morale is strong!
> - Poised to go public next spring!
```

File: content/markdown/concepts/headings/headings.md [hunk 8]
```diff
@@ -60,9 +60,9 @@ For compatibility, always put a space between the number signs and the heading n
 
 🚫 Don't: `#Breaking News`
 
-You should also put blank lines before and after a heading for compatibility and readability.
+Put blank lines before and after a heading for compatibility and readability.
```
Comment (Dusch4593) [inline content/markdown/concepts/headings/headings.md:63]:
```suggestion
Headings should have a blank line above and below for compatibility and readability:
```

File: content/markdown/concepts/tables/tables.md [hunk 9]
```diff
@@ -65,10 +65,10 @@ You can align text in the columns to the left, right, or center by adding a colo
 
 ## Text Formatting
 
-The text can be formatted within tables. For example, you can add links, emphasis, and inline code (words or phrases in backticks only, not code blocks).
+Text can be formatted within tables. For example, links, emphasis, and inline code (words or phrases in backticks only, not code blocks) are all readily available for use within a table.
 
-You can’t add headings, blockquotes, lists, horizontal rules, images, or HTML tags.
+Several formatting options are not availble within tables. Most notably headings, blockquotes, lists, horizontal rules, images, and HTML tags.
```
Comment (Dusch4593) [inline content/markdown/concepts/tables/tables.md:70]:
```suggestion
Several formatting options are not available within tables, including:

- Headings
- Blockquotes
- Horizontal rules
- Images
- Lists
- HTML tags
```

File: content/markdown/concepts/tables/tables.md [hunk 10]
```diff
@@ -65,10 +65,10 @@ You can align text in the columns to the left, right, or center by adding a colo
...
 
 ## Escaping Characters
 
-You can display a vertical line "|" character in a table by using its HTML character code `&#124;`.
+To display a vertical line "|" character in a table, use the HTML character code `&#124;`.
```
Comment (Dusch4593) [inline content/markdown/concepts/tables/tables.md:74]:
```suggestion
Pipe characters "|" can be displayed in a table with the [HTML character code](https://www.codecademy.com/resources/docs/html/entities) `&#124;`.
```

File: content/markdown/concepts/tables/tables.md [hunk 11]
```diff
@@ -36,7 +36,7 @@ The output would look like:
 Cell widths can vary, as shown below.
```
Comment (Dusch4593) [inline content/markdown/concepts/tables/tables.md:36]:
```suggestion
Cell widths can vary, as shown below:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@mikikiv Thanks for the changes! 😄 I left some additional comments/suggestion for you to check out and then after that we'll be about ready to merge.

**Review by yangc95 — APPROVED** [review]
LGTM 👍 Anything else to add here @Dusch4593 ?

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Thank you so much @mikikiv for your contributions to these entries! 😄 🎉 🎊 You can checkout the changes by selecting one of concepts under [Markdown](https://www.codecademy.com/resources/docs/markdown)

**Pull request merged**